README.md

```diff
@@ -5,8 +5,6 @@
         <img alt="PyPI" src="https://img.shields.io/pypi/v/prefect?color=0052FF&labelColor=090422"></a>
     <a href="https://github.com/prefecthq/prefect/" alt="Stars">
         <img src="https://img.shields.io/github/stars/prefecthq/prefect?color=0052FF&labelColor=090422" /></a>
-    <a href="https://pepy.tech/badge/prefect/" alt="Downloads">
```

Comment:
instead of removing this we can adjust the link to point to `https://pypi.org/project/prefect/`

Reply:
pypi is the first badge here already 
![image](https://github.com/user-attachments/assets/ca8dd4bc-5c6b-40be-baee-cd5d8772d0f3)

Reply:
actually i found a different way to get the monthly downloads, which is maybe what you were referring to

Reply:
Sorry I missed this comment - what I meant was we can link the downloads badge to our PyPI project instead of the broken link. Looks like you got it all working properly 👍